features/auth.feature

```diff
@@ -0,0 +1,6 @@
+
+Feature: User authentication functionality
+  Scenario: User with valid credentials can login
+    Given user open main page
+    When user logs in with valid username and password
```

Comment:
better to use other verb than logs in, like 'user enters valid username and password